libs/npm/core/src/components/center/Center.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import * as styled from './styled';
+
+export interface ICenterProps extends React.HtmlHTMLAttributes<HTMLDivElement> {}
+
+/** Simple component used for centering text/items within a cell of a table */
+export const Center: React.FC<ICenterProps> = ({ children }) => {
+  return <styled.Center className="center">{children}</styled.Center>;
```

Comment:
From a maintenance perspective it's overkill and unless we can solve testing the shared library may frustrate us with future changes.  As far as I can tell you're just centering, which can be achieved with a className.  I was expecting the component to do more than just center text.  I don't see a lot of value in this one.  I suppose it saves you writing `<div className="center"></div>`.  I suppose if it did some additional stuff it would make sense.

Reply:
Fair enough, I can take it out!